lib/javascript/service.js

```diff
@@ -4,23 +4,22 @@ class Service {
 
   constructor(endpoint) {
```

Comment:
When we're reconciling our parts of the project today, we should use this constructor in the food.html apr calls too.